### PR TITLE
Activate OVA steps in CI

### LINKED ISSUE #804: Activate existing OVA CI Build pipeline
**User Statement:**

As a developer I want to build the OVA when any of its components has a new artifact.

**Details:**
When any of Admiral, Harbor, Engine, UI plugin push a new artifact, run the OVA build

Pipeline steps are already existing in `drone.yml` file but commented. This issue involves uncommenting those steps, run and resolve any issues, build server (used for packer) assignment.
 
**Acceptance Criteria:**
- [ ] Automated build produces an OVA
- [ ] OVA is uploaded to `vic-product-ova-builds`
- [ ] Existing `dinv` is renamed to DCH

### PR DESCRIPTION
**Pull request by mdharamadas1**
Fixes #804 
Activate existing OVA steps in CI pipeline. Update steps as necessary.

### CODE REVIEW

File: .drone.local.yml [hunk 1]
```diff
@@ -1,52 +1,32 @@
-# After any change to this file you MUST re-sign and checkin the .drone.yml.sig
-# When you are ready to submit a pull request, you must regenerate .drone.yml.sig for the vmware/vic-product repo:
-# $ export DRONE_SERVER=https://ci.vcna.io
-# $ export DRONE_TOKEN=<token>
-# $ drone sign vmware/vic-product
```
Comment (andrewtchin) [inline .drone.local.yml]: Do we not need to do these anymore?

Comment (mdharamadas1) [reply .drone.local.yml]: Yes

File: .drone.yml [hunk 2]
```diff
@@ -41,70 +41,84 @@ pipeline:
...
-#     commands:
-#       - mkdir bundle
-#       - cp bin/vic-*.ova bundle
-#       - ls -al bundle
+  notify-slack:
+    image: plugins/slack
+    webhook: ${SLACK_URL}
+    username: drone
+    template: >
+      build https://ci.vcna.io/vmware/vic-product/{{ build.number }} finished with a {{ build.status }} status
```
Comment (andrewtchin) [inline .drone.yml]: is it possible to add the github username of the build owner? not necessarily for this PR

File: .drone.local.yml [hunk 3]
```diff
@@ -1,52 +1,32 @@
...
   unified-ova-build: 
     image: gcr.io/eminent-nation-87317/vic-product-build:latest
     pull: true
     privileged: true
     environment:
-       DRONE_TIMEOUT: ${DRONE_TIMEOUT}
-       DRONE_TIMEOUT_INACTIVITY: ${DRONE_TIMEOUT_INACTIVITY}
-       DRONE_BUILD_NUMBER: ${DRONE_BUILD_NUMBER}
-       TERM: xterm
+      ADMIRAL: ${ADMIRAL}
```
Comment (andrewtchin) [inline .drone.local.yml]: Where do all these vars get set?

File: .drone.yml (hunk 2 above)
Comment (mdharamadas1) [reply .drone.yml]: Yes, drone already provides `DRONE_REPO_OWNER`

File: .drone.local.yml (hunk 3 above)
Comment (mdharamadas1) [reply .drone.local.yml]: They are set via Drone secrets while executing locally, we yet to expand on that once we figure out how to do release versioning.

File: .drone.yml (hunk 2 above)
Comment (andrewtchin) [reply .drone.yml]: Cool I think that'd be useful here and in vmware/vic repo

Comment (mdharamadas1) [reply .drone.yml]: Agree, I have added #919 and https://github.com/vmware/vic/issues/6542

**Pull request merged**